qa/vagrant/src/test/resources/packaging/tests/example/scripts/is_guide.painless

```diff
@@ -1 +1,5 @@
-_source['title'] == 'Elasticsearch - The Definitive Guide'
+def min = 300;
```

Comment:
OH! I think it is because Elvis can't be assigned to an int, only Integer. It has to do with how painless doesn't do much autoboxing/autounboxing. So `def params.min_num_pages ?:  300` should work. I had a [PR](#21675) that'd make assigning to an `int` work but it added a lot of complexity. I might see if I can revisit it in a way that isn't so complex so we can get this.....